getting_started/step_by_step/scripting_first_script.rst

```diff
@@ -278,6 +326,14 @@ them.
 
     Position += velocity * (float)delta;
 
+ .. code-tab:: cpp C++
+
+    // Note that the directional Vector2 constants do not exist in GDExtension right now. So Vector2(0, -1) must be used.
+    Vector2 velocity = Vector2(0, -1).rotated(get_rotation()) * speed;
```

Comment:
Unless you use `using namespace Godot;` you need to prefix the namespace
```suggestion
    godot::Vector2 velocity = godot::Vector2(0, -1).rotated(get_rotation()) * speed;
```

Reply:
Do you think it would be best for the C++ documentation to use `using namespace godot;`? Or would throwing `godot::` on everything be better? The additional characters from using `godot::` might increase the mental load on reading the docs for begginers for no actual gain but then again `using namespace` is generally discouraged so it's hard what to say is best, up to you.

Reply:
The godot-cpp example uses `using` so I'd use that here too

Reply:
Nice that makes sense, i'll go with `using namespace godot;` then. One last question then i'll commit these tweaks, do you think it would be best to wrap the custom namespace like this:

```cpp
namespace CustomNamespace {

}
```

Or just do it with `using namespace CustomNamespace;` (this is what I do in my extension code, it makes the debug symbols easier to parse through having a custom namespace but idk what would be idomatic)

Or would it be best in the docs to not even worry about the custom namespace at all? Again totally up to you.

Reply:
You shouldn't use the `using` here as you're in the header (if you put the code in a `.cpp` file you should)

Reply:
Gotcha, I'll leave it as is then and commit what I got. Makes sense why you wouldn't want that in the header 👍